sample-apps/go-sample-app/collection/request_metrics.go

```diff
@@ -0,0 +1,113 @@
+package collection
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"sync/atomic"
+
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/instrument/asyncint64"
+	"go.opentelemetry.io/otel/metric/instrument/syncint64"
+)
+
+// requestBasedMetricCollector contains all the request based metric instruments.
+type requestBasedMetricCollector struct {
+	totalBytesSent syncint64.Counter
+	totalRequests  asyncint64.Counter
+	latencyTime    syncint64.Histogram
+	config         Config
+	meter          metric.Meter
+	n              int64
```

Comment:
```suggestion
	n              int64
```
Is there a different variable name that can be used here? I think it would be a best practice to avoid one letter names.

Reply:
You're right. I updated the variable name